flink-formats/flink-parquet/src/main/java/org/apache/flink/formats/parquet/utils/ParquetSchemaConverter.java

```diff
@@ -113,4 +113,13 @@ public static int computeMinBytesForDecimalPrecision(int precision) {
         }
         return numBytes;
     }
+
+    // From DecimalDataUtils
+    public static boolean is32BitDecimal(int precision) {
```

Comment:
Not sure I like this change because we have now duplicated the code and the methods in `DecimalDataUtils` appear unused. WDYT about moving `DecimalDataUtils` to table-commons because I can image other formats also want to use it?

Reply:
> WDYT about moving DecimalDataUtils to table-commons because I can image other formats also want to use it?

I disagree, because we're rather trying to avoid filling table common with utils classes, and hiding them as much as we can. In particular we already tried to move `DecimalDataUtils` to common and then we decided to revert it, because it's not a very well designed class and accesses directly to package private fields of `DecimalData`.

If this is a method that connectors really need, then I suggest we need it in as instance method of `DecimalData`.